src/wallet/Transact.cpp

```diff
@@ -0,0 +1,97 @@
+#include "Transact.h"
```

Comment:
I'm wondering if we can reuse the code from PegOut from normal sends since the code is 95% identical. Perhaps a function that takes both a normal and a stealth address (either can be null). Then if normal is supplied add pegout kernel etc., if stealth is supplied add receiver output etc.

Reply:
In a future PR, I'm going to clean up one of out test framework's TxBuilders, and use that instead. I didn't do that at first because I wanted to have each flow explicitly coded to assist with reviewing and comparing against the actual spec. Once I'm satisfied we have full tests for each flow, we'll switch to the more generic approach.